DQMOffline/Trigger/python/SoftOSMonitor_cff.py

```diff
@@ -5,17 +5,17 @@
 # HLT_PFMETNoMu90_PFMHTNoMu90_IDTight
 DoubleMu3_PFMET50_DZ_PFMHT60_METmonitoring = hltSOSmonitoring.clone()
 DoubleMu3_PFMET50_DZ_PFMHT60_METmonitoring.FolderName = cms.string('HLT/SOS/MET/')
-DoubleMu3_PFMET50_DZ_PFMHT60_METmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DoubleMu3_PFMET50_DZ_PFMHT60_v1")
-DoubleMu3_PFMET50_DZ_PFMHT60_METmonitoring.denGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_v7")
+DoubleMu3_PFMET50_DZ_PFMHT60_METmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_DoubleMu3_PFMET50_DZ_PFMHT60_v*")
```

Comment:
which is the name of the path ?
in the silvio's google doc I find a slightly different naming :(